src/Microsoft.Azure.SignalR.Common/ClientInvocation/ClientResultsManager.cs

```diff
@@ -0,0 +1,323 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#if NET7_0_OR_GREATER
+using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Azure.SignalR.Protocol;
+using System.Buffers;
+using Microsoft.AspNetCore.SignalR.Protocol;
+using Microsoft.AspNetCore.SignalR;
+
+namespace Microsoft.Azure.SignalR
+{
+    internal class ClientResultsManager: IClientResultsManager, IInvocationBinder
```

Comment:
Do we need implement `IInvocationBinder`?

Reply:
Yes, we need. 
It's used in Line 118 (second parameter `this`) `if (proto.TryParseMessage(ref message, this, out var message1))`   to parse message

Reply:
Yes, we need.
It's used in Line 118 `if (proto.TryParseMessage(ref message, this, out var message1))` , second parameter `this`, to parse `ReadOnlySequence<byte> message`